storage/s3_test.go

```diff
@@ -523,6 +523,15 @@ func TestS3CopyEncryptionRequest(t *testing.T) {
 				}
 				assert.Equal(t, aclVal, tc.expectedAcl)
 			})
+			mockApi.Handlers.Unmarshal.PushBack(func(r *request.Request) {
```

Comment:
is this still required, even after the gofakes3 change?

Reply:
yess, for this test we do not interact with gofakes3 but return an empty response with [this](https://github.com/peak/s5cmd/pull/227/files#diff-2ba9e7f40440a8e598a45a16616a9ec760279956b775da2d7dc7b9f393e2e354L501) handler.

Reply:
ah ok.